playground/backend/cmd/server/controller_test.go

```diff
@@ -108,6 +115,10 @@ func teardown(server *grpc.Server) {
 	if err != nil {
 		panic(fmt.Errorf("error during test teardown: %s", err.Error()))
 	}
+	err = os.RemoveAll(javaLogConfigFilename)
```

Comment:
This code repaeting several times: 
err = os.RemoveAll(javaLogConfigFilename)
if err != nil {
		panic(fmt.Errorf("error during test teardown: %s", err.Error()))
	}
Maybe extract it to the method? as arguments folder to remove and error message

Reply:
Changed.